quisby.py

```diff
@@ -280,6 +301,7 @@ def compare_results(args):
 
 def reduce_data():
     data_handler()
+    #compare_results(["1GiqDQY2C5T-llFac5zJwHOwaIxdQzKguRMHEWW1UC0A","1coe4XmL4p3qfPCjjouxOJj4SfiK3zvhm_TVXrH4a404"])
 
```

Comment:
Looks like you left this behind?

Reply:
Ohh wait i will remove this for now. Will push compare changes in next PR.